src/isar/state_machine/states/going_to_lockdown.py

```diff
@@ -54,6 +54,7 @@ def _mission_status_event_handler(
                 if mission_status != MissionStatus.Successful:
                     return state_machine.lockdown_mission_failed  # type: ignore
 
+                state_machine.print_transitions()
```

Comment:
Tbf I'm not sure if we need to print these out anymore. They don't really provide any new information. As far as I can tell, the transition list is only used for the older integration tests.